### PR TITLE
Add eslint-config-react-app

### PR DESCRIPTION
**Pull request by ashinzekene**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ If you want to contribute, please read the [contribution guidelines](contributin
 - [Facebook](https://www.npmjs.com/package/eslint-config-fbjs) - Sharable config for Facebook's style guide.
 - [Google](https://github.com/google/eslint-config-google) - Shareable config for the [Google style](http://google.github.io/styleguide/javascriptguide.xml).
 - [Problems](https://github.com/RyanZim/eslint-config-problems) - Shareable config that only catches actual problems, and doesn't enforce stylistic preferences.
+- [React App](https://github.com/facebook/create-react-app/tree/master/packages/eslint-config-react-app) - Sharable config for [React](https://reactjs.org) projects
```
Comment (dustinspecker) [inline readme.md]:
```suggestion
- [React App](https://github.com/facebook/create-react-app/tree/master/packages/eslint-config-react-app) - Sharable config for [React](https://reactjs.org) projects.
```

Can you add a period at the end please? awesome-lint expects descriptions to end with a period.

**Comment by dustinspecker** [comment]
Thank you! Just a small lint fix and then happy to merge this!

File: readme.md (hunk 1 above)
Comment (ashinzekene) [reply readme.md]: Done

**Comment by dustinspecker** [comment]
Thank you!

**Pull request merged**